.dockerignore

```diff
@@ -3,3 +3,7 @@
 .kube
 .git
 .env
+libs/common/tests/
+services/virtual-assistant/tests/
+services/watson-extension/tests/
```

Comment:
I wonder if `/tests/` or `***/tests/` would catch em all

Reply:
Not sure if `***` is a thing in dockerignore. I think this might work, but not confident -> `test*/`

Reply:
Apparently it is supported:
https://docs.docker.com/build/concepts/context/#syntax

> Beyond Go's filepath.Match rules, Docker also supports a special wildcard string ** that matches any number of directories (including zero). For example, **/*.go excludes all files that end with .go found anywhere in the build context.